test/data/forms.js

```diff
@@ -55,9 +56,11 @@ const forms = dataStore({
       enketoId: !draft ? enketoId : null,
       enketoOnceId,
       state,
-      createdAt: inPast
-        ? fakePastDate([lastCreatedAt, project.createdAt, createdBy.createdAt])
-        : new Date().toISOString(),
+      createdAt: publishedAt != null
```

Comment:
Comment that we're setting publishedAt to createdAt in order to ensure that createdAt is not after publishedAt.